docs/template_extension/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ file(GLOB_RECURSE SRC *.cpp)
 
 add_library(${TARGET_NAME} SHARED ${SRC})
 
+target_include_directories(${TARGET_NAME} PRIVATE ${InferenceEngine_INCLUDE_DIRS})
+
 target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_EXTENSION_API)
-target_link_libraries(${TARGET_NAME} PRIVATE ${InferenceEngine_LIBRARIES}
+target_link_libraries(${TARGET_NAME} PRIVATE IE::inference_engine_legacy IE::inference_engine IE::inference_engine_nn_builder
```

Comment:
`IE::inference_engine_nn_builder` is removed in master branch.

Reply:
I have fixed it.